src/scrypt.rs

```diff
@@ -32,8 +32,8 @@ pub fn derive_key(pass: &Password, salt: &[u8; 32], n: u32, p: u32, r: u32) -> R
     }
 
     let mut derived_key = vec![0u8; KEY_LENGTH];
-    let scrypt_params = ScryptParams::new(log_n, r, p);
-    scrypt(pass.as_bytes(), salt, &scrypt_params, &mut derived_key);
+    let scrypt_params = ScryptParams::new(log_n, r, p).unwrap();
+    let _ = scrypt(pass.as_bytes(), salt, &scrypt_params, &mut derived_key);
```

Comment:
@sgkim126 I need your help. This code need error handling because the output format of `scrypt()` is `Result<(), InvalidOutputLen>`. However, I'm not used to doing error handling yet. I didn’t know whether to use `map_err()` or `unwrap_or_else()` and how to use these, so I wrote _ for the time being.

Reply:
You need to implement type conversion function from the foreign type to the local type. If conversion is possible, you can use `?` operator simply. I'll help you for the problem.